test/es6/proxyenumremoval.js

```diff
@@ -0,0 +1,69 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+var passed = 1;
+passed &= typeof Reflect.enumerate === 'undefined';
+
+var proxy = new Proxy({}, {
+  enumerate: function() {
+    passed = 0;
+  }
+});
+for(var key in proxy);

```

Comment:
done

---
In reply to: [63960080](https://github.com/Microsoft/ChakraCore/pull/976#discussion_r63960080) [](ancestors = 63960080)